ktor-server/ktor-server-cio/jvmAndNix/src/io/ktor/server/cio/backend/HttpServer.kt

```diff
@@ -60,7 +60,11 @@ public fun CoroutineScope.httpServer(
 
             try {
                 while (true) {
-                    val client: Socket = server.accept()
+                    val client: Socket = try {
+                        server.accept()
```

Comment:
Could you tell me why we should ignore any IOExceptions?

I would add some small random delay and counter with limit to prevent dead loop

Reply:
Because now clients can connect and immediately drop the connection, crashing the whole server, like the case with nmap.